web/functions/algolia-search/algolia-search.js

```diff
@@ -62,6 +64,7 @@ exports.handler = function (event, context, cb) {
               downloadURL: doc.downloadURL,
               sourceURL: doc.sourceURL,
             };
+          // Transforming article datatypes
```

Comment:
Let's add blog posts to line 22 (which should share the properties with projects you specified below).

Reply:
If they have the same properties, I'll just keep the switch cases the same and just make a note that the default takes both projects and posts into account